spec/lib/api/2.0/skus-spec.js

```diff
@@ -3,241 +3,263 @@
 
 'use strict';
 
-describe('Http.Api.Skus', function () {
+describe('Http.Api.Skus.2.0', function() {
+    var waterline;
     var workflowApiService;
-    var _;
+    var nodeApiService;
+    var Promise;
+    var Constants;
+    var Errors;
+    var skuApiService;
+
     before('start HTTP server', function () {
         this.timeout(5000);
-        workflowApiService = {
-            createAndRunGraph: sinon.stub()
-        };
-        helper.setupInjector([
-            helper.require("/lib/services/sku-pack-service"),
-            dihelper.simpleWrapper(function() { arguments[1](); }, 'rimraf')
-        ]);
         return helper.startServer([
-            dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
         ]).then(function() {
-            _ = helper.injector.get('_');
+            waterline = helper.injector.get('Services.Waterline');
+            sinon.stub(waterline.nodes);
```

Comment:
These stubs should be restored in your after callback so that subsequent tests do not use them.